log/log.go

```diff
@@ -15,6 +18,10 @@ type Logger interface {
 	Log(keyvals ...interface{}) error
 }
 
+// ErrMissingValue may be appended to keyval slices with an odd length to
+// populate the missing value.
+var ErrMissingValue interface{} = errors.New("(MISSING)")
```

Comment:
- Right now it reads as if users of the package should do the appending, when actually it's the package itself. Can we change the wording, e.g. "ErrMissingValue may be appended by package log to keyval slices with an odd length, to populate the missing value"? — Or, will users even need to introspect for this value? Could this var be unexported?
- Is the explicit interface{} necessary?